<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>290</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>290</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>290</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLCDNumber" name="Time_number">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>481</width>
      <height>211</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>8</number>
    </property>
   </widget>
   <widget class="QPushButton" name="timer_btn">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>10</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Timer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="time_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Time</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Sw_btn">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Stopwatch</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="Sw_number">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>481</width>
      <height>161</height>
     </rect>
    </property>
    <property name="accessibleName">
     <string/>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="digitCount">
     <number>8</number>
    </property>
    <property name="value" stdset="0">
     <double>0.000000000000000</double>
    </property>
    <property name="intValue" stdset="0">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="Sw_Start_btn">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>230</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Sw_Stop_btn">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>230</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Pause</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="Timer_number">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>60</y>
      <width>411</width>
      <height>161</height>
     </rect>
    </property>
    <property name="accessibleName">
     <string/>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="digitCount">
     <number>8</number>
    </property>
    <property name="value" stdset="0">
     <double>0.000000000000000</double>
    </property>
    <property name="intValue" stdset="0">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="Plus_btn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Min_btn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Timer_Start_btn">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>230</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Timer_Stop_btn">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>230</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Pause</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
